### ISSUE TITLE
Having too many npcs spawned crashes the game

### ISSUE THREAD

**Issue by swagmaster360** [JACoders/OpenJK#847]
Reporting a bug? Please make sure you've given the following information - thanks!

**Operating system and version:**
Windows 7 Home Premium

**Is this for single player or multiplayer?**
Singleplayer, as well as probably mp

**Description of the bug (and if possible, steps to reproduce the bug):**
Okay so I was playing movie duels 2 on the geonosis arena duel, and when i got to actually fighting the droids, i couldn't kill them fast enough to prevent this many from spawning in, and because my game was lagging with 4 gigs or ram installed and a decent video card installed, it lagged to where the game crashed, the bug always happens when there are too many npcs spawned, and theres like 4 types of npcs too, the bug didn't happen when i was using the base game to run it

**What did you expect to happen instead?**
OpenJK shouldn't crash with this many npcs lagging the game, especially when the base game didn't have the crash issue.

**Comment by ensiform** [comment]
Ram above ~2 gig isn't usable due to 32 bit process without PAE.

**Comment by swagmaster360** [comment]
this isn't movie battles 2, its movie duels 2, a single-player mod: http://www.moddb.com/mods/star-wars-movie-duels, i think it could be an issue with base game

**Comment by ensiform** [comment]
It's possible.  Sorry for misunderstanding.

**Comment by swagmaster360** [comment]
its alright, perhaps check on the code for npc limits, or maybe the memory usage for the amount of npcs spawned?

**Comment by ensiform** [comment]
A report of where the crash happens (in debug version) with specific reproducible steps would be nice but I'm not expecting you to have that.

**Comment by mrwonko** [comment]
I really need to work on #141, that would make fixing these crashes much easier.

**Comment by swagmaster360** [comment]
oh yeah, to reproduce the bug, play movie duels 2 with openjk (obviously), play the geonosis arena mission, get to the part where after the cutscene you jump off, then the bug will occur with this many npcs coming in

**Comment by ensiform** [comment]
Well is there any error messages or just crash to desktop?

**Comment by swagmaster360** [comment]
crash to desktop

**Comment by xycaleth** [comment]
Probably duplicate of #817.

**Comment by swagmaster360** [comment]
by npc type, do you mean by npc name, or npc class?

**Comment by xycaleth** [comment]
I think it's NPC name.

**Comment by swagmaster360** [comment]
ah yes, there is a lot of npcs with different names, so it is probably #817

**Comment by ensiform** [comment]
Does movie duels 2 use it's own game mod DLL?

**Comment by swagmaster360** [comment]
no, its a total conversion that uses vanilla game, no custom dlls

**Comment by ensiform** [comment]
Just wanted to make sure.

**Comment by swagmaster360** [comment]
well this bug report might not have been totally useless, as #817 is confirmed to be a singleplayer bug as well
